osu.Game/Overlays/Settings/Sections/General/UpdateSettings.cs

```diff
@@ -74,10 +73,33 @@ private void load(Storage storage, OsuConfigManager config, OsuGame game)
                     Action = () => storage.PresentExternally(),
                 });
 
+                Add(new SettingsButton
+                {
+                    Text = "Compress log files",
+                    Keywords = new[] { @"bug", "report", "logs" },
+                    Action = () =>
+                    {
+                        var logStorage = storage.GetStorageForDirectory(@"logs");
```

Comment:
Wish this could use `Logger.Storage` but it's currently `private`. I don't think it would hurt to make the framework-side getter public.

Reply:
Applied this change.